packages/web/components/radio-with-options/radio-with-options.tsx

```diff
@@ -58,7 +62,8 @@ export const RadioWithOptions = ({
           }
           value={value}
         >
-          {label}
+          {/**the following condition checks if the label passed is from an i18n translation path */}
+          {label.includes(".") ? t(label) : label}
```

Comment:
I will mention it can be possible to not have a "." in a translation path.

Reply:
Got it! Do you think it will be better if I pass a flag as a prop that determines the presence of a label that represents an i18n path? (e.g. `labelIsI18nPath: boolean`)

Reply:
We should remove `useTranslation` from radio-with-options, if not, the component is not generic

Reply:
We removed useTranslation inside the component to fix this problem and also improve the component